documentation/asciidoc/accessories/camera/camera_usage.adoc

```diff
@@ -20,6 +20,8 @@ Users are encouraged to use the newest OS images and the `libcamera`-based stack
 * Nearly all aspects of the camera tuning can be changed by users.
 * It integrates much more conveniently with other standard Linux APIs.
 * Raspberry Pi supply a set of `libcamera-apps` which emulate most of the features of the legacy applications.
+* It provdes a feature-rich post-processing framework integrating OpenCV and TensorFlow Lite.
+* Libcamera enables much more straightforward control of sensor and 3A parameters.
```

Comment:
3A ??

Reply:
Ah, I tend to forget that the whole world does not understand that. I mean, how can they not know? But I will change it for them!